### DISCUSSION TITLE
Busco Team para el Hackaton

### DISCUSSION THREAD

**Discussion by gumodi35** [Launch-X-Latam/MisionFrontEnd#501] [Team Posts]
Si alguien busca integrante para su equipo mande mensaje, busco unirme a un equipo para el Hackaton!

**Comment by EliasQuino** [comment]
Para mi tambien

**Comment by JCCotrina** [comment]
Hola tambien estoy interesado formar un equipo.

**Comment by sudoagile** [comment]
Estimados, muchos correos, no deberian usar el correo para mensajes simples, para eso existen otras soluciones

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo y me interesa formar equipo
correo: lx11002@innovaccion.mx
